Content.Server/_Goobstation/StationEvents/Metric/Components/AnomalyMetricComponent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Shared.FixedPoint;
+
+namespace Content.Server._Goobstation.StationEvents.Metric.Components;
+
+[RegisterComponent, Access(typeof(AnomalyMetric))]
+public sealed partial class AnomalyMetricComponent : Component
+{
+    /// <summary>
+    ///   Cost of a growing anomaly
+    /// </summary>
+    [DataField("growingCost")]
```

Comment:
`Data field GrowingCost in data definition AnomalyMetricComponent has an explicitly set tag that matches autogenerated tag`
Same for all the others in this file
```suggestion
    [DataField]
```